### DISCUSSION TITLE
Fix for error in Postgres healthcheck

### DISCUSSION THREAD

**Discussion by husneck** [outline/outline#6768] [Self Hosting]
There is a small issue in the docker-compose.yml file in the installation guidelines: https://docs.getoutline.com/s/hosting/doc/docker-7pfeLP5a8t

Line 37: `test: ["CMD", "pg_isready -d outline -U user"]`
should be: `test: ["CMD", "pg_isready", "-d", "outline", "-U", "user"]`

**Comment by tommoor** [comment]
This is functionally the same, no? Are you seeing issues with it running

**Reply by husneck** [reply]
no issues running it, but healtchcheck issues without the extra quotation marks
![Screenshot 2024-04-05 133832](https://github.com/outline/outline/assets/5699950/815ceab6-b114-49cf-b806-8b4d5167f2c6)

**Reply by tommoor** [reply]
Fixed, thanks for raising
